unison-cli/src/Unison/Codebase/Editor/HandleInput.hs

```diff
@@ -1775,6 +1786,75 @@ handleGist :: GistInput -> Cli ()
 handleGist (GistInput repo) =
   doPushRemoteBranch (GistyPush repo) Path.relativeEmpty' SyncMode.ShortCircuit
 
+handleDiffNamespaceToPatch :: Text -> DiffNamespaceToPatchInput -> Cli ()
+handleDiffNamespaceToPatch description input = do
+  Cli.Env {codebase} <- ask
+
+  branch1 <- Branch.head <$> Cli.resolveBranchId (input ^. #branchId1)
+  branch2 <- Branch.head <$> Cli.resolveBranchId (input ^. #branchId2)
+  branchDiff <- liftIO (BranchDiff.diff0 branch1 branch2)
+
+  -- Given {old referents} and {new referents}, create term edit patch entries as follows:
+  --
+  --   * If the {new referents} is a singleton set {new referent}, proceed. (Otherwise, the patch we might create would
```

Comment:
In which cases is `new referents` not a singleton set, and what can we do in that case?

Reply:
I believe we can also hit that case if we have something like:

|name|old|new|
|----|---|---|
|name1|#abc|#def|
|alias|#abc|#ghi|

Which I think is also pretty rare, but would probably nice to see a:

> Reference #abc maps to more than one new term (new1, new2) and couldn't be added to the patch

Or something like that.
Could probably be added in a separate PR though if it's annoying.

Reply:
Yeah, I think doing something other than ignoring this case should just be its own ticket and PR.

Reply:
The example you gave is indeed different, though. I think this PR, as written, would just add both of those entries to the patch, and then the patch would be borked.